nss-cohort-40/bangazon-ecommerce-api-lumberjacks-back
# Add payment type

# Description
Added profile page, affordances, and form for the user to add a payment type.

Fixes #7 (https://github.com/nss-cohort-40/bangazon-ecommerce-api-lumberjacks-back/issues/7)

## Type of change
Please delete options that are not relevant.

# Testing Instructions
- Make sure you're logged in
- Click the 'Profile' page from the nav bar. Make sure customer info is populated.
- Click 'Add Payment Option' affordance
- Complete form and submit
- Check DB to ensure payment type was added

# Checklist:
